designs/administrator/future-openshift/list-descriptions/index.md

```diff
@@ -0,0 +1,27 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Adding resource description for resource lists
+
+Content from the Explore page can be surfaced by placing resource descriptions within context.
+Resources lists can be enhanced by adding resource descriptions, pulled from the "Explore" page.
+This can save the user some time as well as exposing the user to further content that was somewhat hidden.
+
+## Resource list page
+![Deployment page](img/List-Description-00.png)
```

Comment:
deployment is a good example of a short description that fits well. What would a long description look like?

Reply:
Added a cap of 120 characters that will break to "...Read more"